recipes-ni/initscripts-nilrt/files/nisetupdump

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+# (c) 2013 National Instruments Corporation
+[ "${VERBOSE}" != "no" ] && echo -n "Configuring core dump settings:"
+
+safemodeflag=/etc/natinst/safemode
+
+#Create core dump files in the root directory of runmode with the
+#executable name as the suffix.
+if [ -f $safemodeflag ]; then
+	echo "/mnt/userfs/var/local/natinst/log/core_dump.%E" > /proc/sys/kernel/core_pattern
+else
+	echo "/var/local/natinst/log/core_dump.%E" > /proc/sys/kernel/core_pattern
+fi
+
+#Enable core dump for tainted binaries
+echo 1 > /proc/sys/fs/suid_dumpable
```

Comment:
We need to keep these settings*, but the rest of this script is only relevant to safemode. Surely we already have a package in OE that sets `ulimit` or something? That's probably a better place to put sysctl options like this.

* Though the `core_pattern` should be set to a better location. Where should that be @gratian ?

Reply:
There is [this](https://github.com/ni/meta-nilrt/blob/nilrt/master/sumo/recipes-ni/base-files-nilrt/files/ulimit.sh#L1) but it gets installed to /etc/profile.d, which is probably not where we want to add this.
How about [nisetupkernelconfig](https://github.com/ni/meta-nilrt/blob/nilrt/master/sumo/recipes-ni/initscripts-nilrt/files/nisetupkernelconfig#L1)?

Reply:
Adding core_pattern also to nisetupkernelconfig for now

Reply:
Yeah, I think moving the core_pattern  into nisetupkernelconfig is OK.